imports/collections/schemas/tags.js

```diff
@@ -88,6 +88,10 @@ export const Tag = new SimpleSchema({
   "displayTitle": {
     type: String,
     optional: true
+  },
+  "active": {
+    type: Boolean,
+    optional: true
```

Comment:
@mikemurray Should we require `active` to be set, and set it as `false` by default? And does this mean.. we have to make a migration for all past Tags and set them as `active` = `true`?

Reply:
Yes, `active` should be required, and we would need a migration to set all previous tags to active.

Reply:
One observation though, it seems we have a few different "enabled/visible" status fields like  `enabled`, `active`, `isVisible`, that we may need to start considering what the status of an "enabled" thing is vs something that's "hidden". Are they the same thing or different?

If nothing else, it should be `isActive` or `isEnabled`

Reply:
Made the name change to `isActive` + added a Migration!